cabal.project

```diff
@@ -30,7 +30,7 @@ packages:
          ./plugins/hls-qualify-imported-names-plugin
          ./plugins/hls-code-range-plugin
          ./plugins/hls-change-type-signature-plugin
-         ./plugins/hls-stan-plugin
+         -- ./plugins/hls-stan-plugin
```

Comment:
hls-stan-plugin is breaking my builds: https://github.com/haskell/haskell-language-server/actions/runs/5893488060/job/15985146763?pr=3764

https://github.com/haskell/haskell-language-server/issues/3701